code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -301,6 +302,8 @@
 			M << "That name is reserved."
 			return nukelastname(M)
 
+	ticker.mode:last_name = capitalize(newname)
```

Comment:
colons are unsafe, typecast a var and assign ticker.mode to it, istype it and then do this